pshtt.py

```diff
@@ -37,8 +37,8 @@
 # The fields we're collecting, will be keys in JSON and
 # column headers in CSV.
 HEADERS = [
-    "Domain", "Canonical URL", "Live", "Redirect",
-    "Valid HTTPS", "Defaults HTTPS", "Downgrades HTTPS", "Strictly Forces HTTPS",
+    "Domain", "Canonical URL", "Live", "Redirect", "Redirect To",
+    "Valid HTTPS", "Defaults to HTTPS", "Downgrades HTTPS", "Strictly Forces HTTPS",
```

Comment:
I added a `Redirect To` field as explained in the description, and also renamed a field to `Defaults to HTTPS` to make it the same as the field name in previous CSVs.